features-json/spdy.json

```diff
@@ -329,7 +331,8 @@
   },
   "notes":"See also support for [HTTP2](https://caniuse.com/#feat=http2), successor of SPDY.\r\n\r\nChrome will be removing support in 2016 (in favor of HTTP2), other browsers likely to follow.",
   "notes_by_num":{
-    
+    "1":"Deprecated as of macOS Mojave 10.4.4. To be completly removed in a future version of macOS.",
+    "2":"Deprecated as of iOS 12.2. To be completly removed in a future version of iOS."
```

Comment:
```suggestion
    "2":"Deprecated as of iOS 12.2. To be completely removed in a future version of iOS."
```

Reply:
Good catch. Thanks!

Reply:
You need to commit the change to appear live @daytonlowell